app/controllers/collaborators_controller.rb

```diff
@@ -0,0 +1,63 @@
+class CollaboratorsController < ApplicationController
```

Comment:
Look for the indentations through out the files. It's better for readability of code.

Reply:
okay